netebpfext/net_ebpf_ext_program_info.h

```diff
@@ -7,14 +7,17 @@
 #include "ebpf_program_types.h"
 #include "ebpf_shared_framework.h"
 
+#define TRUE 1
+
 #define XDP_EXT_HELPER_FUNCTION_START EBPF_MAX_GENERAL_HELPER_FUNCTION
 
 // XDP helper function prototype descriptors.
 static const ebpf_helper_function_prototype_t _xdp_ebpf_extension_helper_function_prototype[] = {
     {XDP_EXT_HELPER_FUNCTION_START + 1,
      "bpf_xdp_adjust_head",
      EBPF_RETURN_TYPE_INTEGER,
-     {EBPF_ARGUMENT_TYPE_PTR_TO_CTX, EBPF_ARGUMENT_TYPE_ANYTHING}}};
+     {EBPF_ARGUMENT_TYPE_PTR_TO_CTX, EBPF_ARGUMENT_TYPE_ANYTHING},
+     TRUE}};
```

Comment:
`TRUE` isn't very readable here as it doesn't tell the viewer what it means.
Maybe rename to "REALLOCATES_PACKET" or similar.